### PR TITLE
Added Swedish localisation for quick-lint-js

### PR DESCRIPTION
**Pull request by Lindenbyte**
Submitting my pull request Swedish translation/localisation for quick-lint-js. Felt as a something I would be able to start with as a  contribution to the project, even though the value might not be of the greatest significance.

Assuming most people and/or maintainers don't speak Swedish I will say that I tried to stay as close as I could to the message and context given by the English messages.

### CODE REVIEW

**Comment by Lindenbyte** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

File: po/sv_SE.po [hunk 1]
```diff
@@ -0,0 +1,758 @@
+# Copyright (C) 2020  Matthew "strager" Glazar
```
Comment (strager) [inline po/sv_SE.po:1]: Please run tools/compile-translations to compile the translations.

File: po/sv_SE.po [hunk 2]
```diff
@@ -0,0 +1,758 @@
...
+# See end of file for extended copyright information.
```
Comment (strager) [inline po/sv_SE.po:2]: The extended copyright information is missing from the end of the file. Please add it.

**Review by strager — CHANGES_REQUESTED** [review]

**Comment by strager** [comment]
Jag är en tekanna.

**Comment by Lindenbyte** [comment]
Alright, added extended copyright information at end of file and compiled the translations.

And:
Ja du är en tekanna!

**Review by strager — APPROVED** [review]
It works!

```
$ LANGUAGE=sv_SE ./build-debug/quick-lint-js hello.js
hello.js:1:1: warning: användning av odeklarerad variabel: banana [E057]
```

**Comment by strager** [comment]
recheck

**Comment by Lindenbyte** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

**Pull request merged**